packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -61,11 +61,11 @@
 
 /* shared styles used in PlanDetails and PlanUpgradeDetails */
 .accordion-btn {
-  @apply bg-transparent border-transparent cursor-pointer text-blue-500 leading-5 my-2 outline-none py-2 pr-2 pl-6 relative w-auto h-auto focus:border focus:border-solid focus:border-blue-400 focus:py-2 focus:pr-4 focus:pl-10 focus:shadow-none;
+  @apply bg-transparent border-transparent cursor-pointer text-blue-500 leading-5 my-2 py-2 pr-2 pl-6 relative focus:border focus:border-solid focus:border-blue-400 focus:py-2 focus:pr-4 focus:pl-10 focus:shadow-none;
 }
 
 .arrow {
-  @apply before:bg-[url('../components/PlanDetails/images/chevron.svg')] before:content-[''] before:absolute before:h-4 before:w-6 before:top-2.5  before:left-0 focus:before:left-3.5;
+  @apply before:bg-[url('../components/PlanDetails/images/chevron.svg')] before:content-[''] before:absolute before:h-4 before:w-6 before:top-[0.6rem]  before:left-0 focus:before:left-[0.87rem];
```

Comment:
I'm confused.
https://tailwindcss.com/docs/top-right-bottom-left says `top-2.5` is `top: 0.625rem; /* 10px */`. Isn't that "close enough"(tm) to 0.6 that we can avoid hardcoding 0.6rem?
Same w/ `left-[0.87rem]` versus built-in `left-3.5` (which is `left: 0.875rem; /* 14px */`).

Reply:
It does slightly change the appearance - we will need to redesign this whole section/button at a later time, as we are currently blocked from making a number of improvements. The redesign will remove all of this nonsense. :)